src/app/rundown-view/services/action-trigger-producer-key-binding.service.ts

```diff
@@ -277,13 +280,18 @@ export class ActionTriggerProducerKeyBindingService implements KeyBindingService
   }
 
   public destroy(): void {
+    this.systemKeyBindings = []
     this.keyBindingsSubject.complete()
+    this.keyBindingsSubject = new BehaviorSubject(this.systemKeyBindings)
```

Comment:
```suggestion
    this.keyBindingsSubject.next(this.systemKeyBindings)
```

I don't know what purpose it serves to call complete to then directly update it afterwards 😄 Will this change do?

Reply:
The purpose is that when we call destroy, we should ensure that updates are no longer propagated to old consumers.